src/main/java/com/teamacronymcoders/epos/feats/ObsidianSmasherFeat.java

```diff
@@ -0,0 +1,43 @@
+package com.teamacronymcoders.epos.feats;
+
+import com.teamacronymcoders.epos.api.EposAPI;
+import com.teamacronymcoders.epos.api.feat.Feat;
+import com.teamacronymcoders.epos.api.feat.FeatBuilder;
+import net.minecraft.block.Blocks;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemTier;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.ToolType;
+import net.minecraftforge.event.entity.player.PlayerEvent;
+
+public class ObsidianSmasherFeat {
+    public static final ResourceLocation TOOL_NAME = new ResourceLocation(EposAPI.ID, "obsidian_smasher");
+    public static final ResourceLocation NO_TOOL_NAME = new ResourceLocation(EposAPI.ID, "obsidian_smasher_no_tool");
+
+    public static final Feat TOOL_FEAT =
+            FeatBuilder.start(TOOL_NAME)
+                    .withEventHandler(PlayerEvent.BreakSpeed.class, (breakSpeed, livingEntity, iCharacterStats) -> {
+                        PlayerEntity playerEntity = breakSpeed.getEntityPlayer();
+                        if (breakSpeed.getState().getBlock() == Blocks.OBSIDIAN &&
+                                playerEntity.getHeldItemMainhand().getHarvestLevel(ToolType.PICKAXE, playerEntity, breakSpeed.getState()) >= ItemTier.DIAMOND.getHarvestLevel()) {
```

Comment:
This is needed since otherwise It'll give a 10x speed modifier to anything trying to break obsidian, the point of the tool-less is for Monks who can mine with an open hand.

Reply:
I more mean I  don't remember if you say "had" a pickaxe and no longer do/something sets the stack size to 0, then isEmpty() would return true. I can't remember if forge would still say that it is a pickaxe though. I meant if it would say it is a pickaxe do we need a `!isEmpty` check

Reply:
It won't it'd turn into ItemStack.EMPTY which is a seperate ItemStack with no ToolType values.

Reply:
So it wouldn't "remember" that it used to be a pickaxe